### DISCUSSION TITLE
Object locking filesystem level protection

### DISCUSSION THREAD

**Discussion by allahshukur-ahmadzada** [minio/minio#20373] [Q&A] [answered]
Hello, I see MinIO has [object locking](https://min.io/docs/minio/linux/administration/object-management/object-retention.html) feature to meet different compliance requirements. I wonder if there are some features also to avoid deletion of files on filesystem level. I know filesystem is out of scope of MinIO maybe but things like hiding file/folder names could help to avoid someone potentially removing a particular file/folder. Is it possible to hide file/folder names on FIlesystem level? encryption helps with that?

**Comment by klauspost** [comment, marked as answer]
No.
